.trivyignore

```diff
@@ -67,4 +67,14 @@ CVE-2024-27304
 # This is resolved in versions of Gloo Gateway that rely on Go1.22 and above (1.17, 1.18)
 # For earlier versions of Gloo Gateway, we confirmed that the vulnerability is not exploitable
 # and captured our findings here: https://github.com/solo-io/solo-projects/issues/7157#issuecomment-2463252858
-CVE-2022-30635
\ No newline at end of file
+CVE-2022-30635
+
+# https://github.com/advisories/GHSA-2mj3-vfvx-fc43
+# https://github.com/advisories/GHSA-gh5c-3h97-2f3q
+# These are not expected to impact us and are difficult to resolve due to breaking API changes that impact our
+# code
+# While this has been resolved on v1.16+, backporting it to lower versions is complicated and we opted to skip it
```

Comment:
should it be resolved in 1.17+ since we are reverting the fix in EE 1.16?

Reply:
yup